pkg/models/temp_user.go

```diff
@@ -82,24 +82,44 @@ type GetTempUsersQuery struct {
 }
 
 type GetTempUserByCodeQuery struct {
-	Code string
-
+	Code   string
 	Result *TempUserDTO
 }
 
+type Time struct{ time.Time }
 type TempUserDTO struct {
-	Id             int64          `json:"id"`
-	OrgId          int64          `json:"orgId"`
-	Name           string         `json:"name"`
-	Email          string         `json:"email"`
-	Role           org.RoleType   `json:"role"`
-	InvitedByLogin string         `json:"invitedByLogin"`
-	InvitedByEmail string         `json:"invitedByEmail"`
-	InvitedByName  string         `json:"invitedByName"`
-	Code           string         `json:"code"`
-	Status         TempUserStatus `json:"status"`
-	Url            string         `json:"url"`
-	EmailSent      bool           `json:"emailSent"`
-	EmailSentOn    time.Time      `json:"emailSentOn"`
-	Created        time.Time      `json:"createdOn"`
+	Id             int64          `json:"id" db:"id"`
+	OrgId          int64          `json:"orgId" db:"org_id"`
+	Name           string         `json:"name" db:"name"`
+	Email          string         `json:"email" db:"email"`
+	Role           org.RoleType   `json:"role" db:"role"`
+	InvitedByLogin *string        `json:"invitedByLogin" db:"invited_by_login"`
+	InvitedByEmail *string        `json:"invitedByEmail" db:"invited_by_email"`
+	InvitedByName  *string        `json:"invitedByName" db:"invited_by_name"`
+	Code           string         `json:"code" db:"code"`
+	Status         TempUserStatus `json:"status" db:"status"`
+	Url            string         `json:"url" db:"url"`
+	EmailSent      bool           `json:"emailSent" db:"email_sent"`
+	EmailSentOn    *time.Time     `json:"emailSentOn" db:"email_sent_on"`
+	Created        Time           `json:"createdOn" db:"created"`
```

Comment:
The more I look at this, the more concerned I am about the `Time` type in the DTO. 

I think it is excellent for handling the database conversion, but we ought to keep all the time-related fields in the DTOs consistent time.Time types. We could create a custom `sqlxTempUser `type, or add an `int64` field  (`IntTime`? `CreatedAtInt`? naming things is hard, you don't have to use my suggestions) to the DTO and populate the `Created` field before returning the value.

What do you think?